ow_plexil/src/plans/IdentifySampleLocationDemo.plp

```diff
@@ -0,0 +1,62 @@
+// The Notices and Disclaimers for Ocean Worlds Autonomy Testbed for Exploration
```

Comment:
Why wouldn't The `IdentifySampleLocationDemo.plp` use the `IdentifySampleTarget.plp`?

Reply:
I originally was trying to prevent taking pictures and pan/tilting from happening twice, but I think that your suggestion is better since readability and re-usability are probably more important on a demo procedure. I changed it so that IdentifySampleLocationDemo now reuses IdentifySampleTarget for both runs.

Reply:
It is not a big deal to re-take pictures every time you need to analyze the site. Unless you want to conserve energy (by avoiding to move the antenna) but we can leave such optimizations to the end-users of autonomy to solve.